modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -1693,6 +1693,13 @@ public void testPostObjectEntry() throws Exception {
 			).toString(),
 			jsonObject.toString(), JSONCompareMode.LENIENT);
 
+		JSONObject attachmentPropertyJSONObject = jsonObject.getJSONObject(
```

Comment:
Let me explain both cases:
- In the attachment field, the response returned by the API call is a JSON Object which contains many properties such as for example its href, creation date... Since the only thing we're introducing to create the field is the attachment ID, I decided to access inside of the JSON Object outputed and check only for the ID.
- In the other cases, I've seen some weird behavior related to Java roundings and in general, JSON Assert not recognizing properly when some number variables have the same value. That's why I decided to check them individually with a very low delta between the expected number and the real number returned by the response.